jumpscale/core/base/fields.py

```diff
@@ -118,6 +118,23 @@ def preprocess(self, value):
         # so, from_raw and to_raw are used only for serialization
         return value
 
+    def validate_named(self, value, name):
+        """
+        validates the value using validate method and appends the field name
+        in case an exxception was thrown
+
+        Args:
+            value (any): in case value is not valid
+            name  (str): the field name
+        Raises:
+            ValidationError: [description]
+
+        """
+        try:
+            self.validate(value)
+        except ValidationError as e:
+            raise ValidationError(str(e) + f" for the field {name}")
```

Comment:
I think the message can be `<field name>: <origianl error message`, it's shorter, right?